install/src/main/java/org/jetbrains/bsp/bazel/install/ProjectViewCLiOptionsProvider.kt

```diff
@@ -45,6 +47,16 @@ object ProjectViewCLiOptionsProvider {
         return ProjectViewTargetsSection(includedTargets, excludedTargets)
     }
 
+    private fun toDirectoriesSection(projectViewCliOptions: ProjectViewCliOptions?): ProjectViewDirectoriesSection? =
+            projectViewCliOptions?.directories?.let(::toDirectoriesSectionNotNull)
+
+    private fun toDirectoriesSectionNotNull(directories: List<String>?): ProjectViewDirectoriesSection {
+        val includedDirectories = calculateIncludedTargets(directories)
+        val excludedDirectories = calculateExcludedTargets(directories)
+
+        return ProjectViewDirectoriesSection(includedDirectories, excludedDirectories)
+    }
+
     private fun calculateIncludedTargets(targets: List<String>?): List<BuildTargetIdentifier> =
```

Comment:
then u can remove `?` here as well and `.orEmpty()`

also, probably the name could be changed - `calculateIncludedValues` or something like that (same for excluded) - it could return `List<String>` and the mapping to `BuildTargetIdentifier` could be done later for targets section, and for dirs it would be a mapping to `Path` (probably)

Reply:
Fixed